condor/condor_meter

```diff
@@ -711,7 +710,19 @@ def classadToJUR(classad):
 
     setIfExists(r.Queue, classad, "JobUniverse", "Condor's JobUniverse field", setstr=True)
     setIfExists(r.NodeCount, classad, 'MaxHosts', "max")
-
+    ########################################################################################
+    ########################################################################################
+    # Code added to send to arbitrary Ads SOFTWARE-2714
+    ArbitraryJobAds = GratiaCore.Config.getConfigAttribute("ArbitraryJobAdAttributes")
+    DebugPrint(5, "ArbitraryJobAd: %s" % str(ArbitraryJobAds))
+    Arbirtray_adlist = ArbitraryJobAds.split(",")
+    DebugPrint(5, "Going through list: %s " % Arbirtray_adlist)
+    for arbitaryAd in  Arbirtray_adlist:
```

Comment:
Variable name `Arbirtray_adlist` is misspelled.  There's also a mix of CamelCase and underscores - looking at the nearby code, stick with CamelCase.